src/Microsoft.SqlTools.ServiceLayer/EditData/Contracts/EditCreateRowRequest.cs

```diff
@@ -0,0 +1,39 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.ServiceLayer.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.EditData.Contracts
+{
+    /// <summary>
+    /// Parameters for the update cell request
+    /// </summary>
+    public class EditCreateRowParams
```

Comment:
Could we have a generic EditParams class with the OwnerUri and have others inherit from it? This would make handling things like the HandleSessionRequest method in the service (that I suggested) easier, since we'd know any params for an edit would have that.